src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+logo.addEventListener(
+  'click',
+  () => {
+    const promise1 = new Promise((resolve, reject) => {
+      resolve();
+    });
+
+    promise1.then(() => {
+      const resolveDiv = document.createElement('div');
+
+      resolveDiv.classList.add('message');
+      resolveDiv.textContent = 'Promise was resolved!';
+      body.append(resolveDiv);
+    });
+  },
+  { once: true },
+);
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
```

Comment:
It's generally a good practice to provide an error message or object when rejecting a promise. Consider passing an error message to `reject()`, like `reject('Error message')`, to provide more context in the `catch` block.